metadata-jobs/mae-consumer/src/main/java/com/linkedin/metadata/kafka/MetadataChangeLogProcessor.java

```diff
@@ -51,18 +49,60 @@ public MetadataChangeLogProcessor(
       @Nonnull final EntityChangeEventGeneratorHook entityChangeEventHook,
       @Nonnull final SiblingAssociationHook siblingAssociationHook
   ) {
-    this.hooks = ImmutableList.of(updateIndicesHook, ingestionSchedulerHook, entityChangeEventHook, siblingAssociationHook);
-    this.hooks.forEach(MetadataChangeLogHook::init);
```

Comment:
I am not seeing where the init() method is being invoked after these changes. Is that method being deprecated? If so, I'd expect to it being removed from the interface or I'd expect to see the init() method invoked on each of the `hookMap` values.

Reply:
Sorry, my bad. I will revert the init call.